app/views/devise/passwords/new.html.erb

```diff
@@ -1,16 +1,21 @@
-<h2>Forgot your password?</h2>
+<div class="text-center reset__password-content">
+  <div class="reset__password-box">
+    <%= image_tag "reset-password.svg", alt: "icon-reset-password", class:"image-reset-password" %>
+    <h2>Forgot your password?</h2>
 
-<%= form_for(resource, as: resource_name, url: password_path(resource_name), html: { method: :post }) do |f| %>
-  <%= render "devise/shared/error_messages", resource: resource %>
+    <%= form_for(resource, as: resource_name, url: password_path(resource_name), html: { method: :post }) do |f| %>
+      <%= render "devise/shared/error_messages", resource: resource %>
 
-  <div class="field">
-    <%= f.label :email %><br />
-    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
-  </div>
+      <div class="field">
```

Comment:
This input would be have a better classname to be more specific what it is. What do tou think?